src/main/resources/application.properties

```
@@ -1 +1,10 @@
-
+databases.users.database.dialect=${USER_DB_DIALECT:org.hibernate.dialect.MariaDBDialect}
+databases.users.database.driver=${USER_DB_DRIVER:com.mysql.cj.jdbc.Driver}
+databases.users.database.url=${USER_DB_URL:localhost}
+databases.users.user.name=${USER_DB_USER_NAME:myusername}
+databases.users.user.password=${USER_DB_USER_PW:astrongpassphrase!}
+encryption.secret=${ENCRYPTION_KEY:12345678901234}
+encryption.salt=${ENCRYPTION_SALT:123456789!1234567}
+service.officer.name=${SERVICE_OFFICER_NAME:ChuckNorris}
+service.officer.password=${SERVICE_OFFICER_PASSWORD:astrongpassphrase!}
+hibernate.dialect=org.hibernate.dialect.MariaDBDialect
```

would we need a newline here too?
```suggestion
hibernate.dialect=org.hibernate.dialect.MariaDBDialect

```